test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -44,8 +44,10 @@ Describe "PSReadLine" -tags "CI" {
 
     It "Should ignore case when using Function binding" {
         Set-PSReadlineOption -EditMode Windows
-        Set-PSReadlineKeyHandler "Ctrl+k" -Function yank # NOTE: lower case function name
-        (Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should Be Yank
+        $lowerCaseFunctionName = "yank";
+        $pascalCaseFunctionName = "Yank"
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function $lowerCaseFunctionName
+        (Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should Be $pascalCaseFunctionName
```

Comment:
OK, perhaps this is an opportunity to clean up other parts of the code. How does this sound:
- instead of using the input (e.g. "yank", "yAnk", "YANK"), we should use methodinfo.name
- briefdescription/longdescription seems not be copied to the relevant metadata fields on the binding when using set-psreadlinekeyhandler with a built-in function binding, I should ensure theses are propagated
- write tests for the above

I was just targeting the bug before. Screw it, let me address all the issues and amend the original issue #4300 to mention the metadata issues. Thoughts?